docs/src/app/new/(content)/components/dialog/demo-hero/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as CssModules } from './css-modules';
+export { default as Tailwind } from './tailwind';
```

Comment:
We'll need to do an `index.ts` file for each demo folder so that it can be automatically imported into the MDX file